src/main/resources/application.properties

```diff
@@ -1,2 +1,8 @@
-# Configuration file
-# key = value
\ No newline at end of file
+# Quarkus logging
+quarkus.log.level       = INFO
+
+# BaseX configuration
+basex.host              = localhost
+basex.port              = 1984
+basex.username          = admin
```

Comment:
Does this need to be stored in a secrets store?

Reply:
> Does this need to be stored in a secrets store?

Indeed, we should avoid committing such information.

Reply:
Ah yes, normally it should be avoided indeed!
This was just for the MvP, these are the defaults.

I can try to use a Secure Store for this!

Reply:
Ah yes, normally it should be avoided indeed!
This was just for the MvP, these are the defaults.

I can try to use a Secure Store for this!

Reply:
I think, it make sense to do so.

Reply:
Of course we can imagine to use secret mechnism such as Vault, or k8s secrets :)

But for the development side we can continue like this :)

Reply:
I made it work with the use of a HasiCorp Vault (extra Docker container that's running).
So that's an option :)

Reply:
Is it okay to keep it like this for now?
(It's still a small MvP, username/password are default).

Or is it necessary right now to use something like a Vault for credentials?

Reply:
https://github.com/com-pas/compas-cim-mapping/issues/14